src/transforms/filter.js

```diff
@@ -16,11 +25,11 @@ exports.name = 'filter';
 exports.attributes = {
     operation: {
         valType: 'enumerated',
-        values: ['=', '<', '>'],
+        values: ['=', '<', '>', 'within', 'notwithin', 'in', 'notin'],
         dflt: '='
     },
     value: {
-        valType: 'number',
+        valType: 'any',
         dflt: 0
     },
     filtersrc: {
```

Comment:
Can we add `identifier` to the `filtersrc` enum? The previous blocker was the animation PR getting merged, but now that it's merged, it's probably a good idea. Without that, I believe @timelyportfolio had to use floating point identity in order to use the lasso selector and then filter to the selected set. Since `in` uses indexOf, I think this perhaps really is a one-liner.